photolog/__init__.py

```diff
@@ -3,7 +3,7 @@
 from flask_wtf.csrf import CsrfProtect
 
 
-photolog = Flask(__name__)
+photolog = Flask(__name__, static_folder='../public')
```

Comment:
Putting `static_folder='...'` into config.py as `STATIC_FOLDER = '...'` didn't seem to work. I wanna try and figure out _why_, but maybe later.